seleniumByTechnocredits/src/seleniumPractise/amarja/ExcelsheetModular.java

```diff
@@ -0,0 +1,84 @@
+package seleniumPractise.amarja;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public class ExcelsheetModular {
+	Workbook myworkbook=null;
+	public ExcelsheetModular(String path) throws IOException
+	{
+		File file =new File(path);
+		FileInputStream input=new FileInputStream(file);
+		myworkbook=new XSSFWorkbook(input);
+	}
+	public String[][] getRow(String sheetName,int rowIndex)
```

Comment:
there is no need to write getRow method as you have already written method for getAllRows.